cmd/kubeadm/app/cmd/options/constant.go

```diff
@@ -16,104 +16,112 @@ limitations under the License.
 
 package options
 
-// APIServerAdvertiseAddress flag sets the IP address the API Server will advertise it's listening on. Specify '0.0.0.0' to use the address of the default network interface.
-const APIServerAdvertiseAddress = "apiserver-advertise-address"
+const (
+	// APIServerAdvertiseAddress flag sets the IP address the API Server will advertise it's listening on. Specify '0.0.0.0' to use the address of the default network interface.
+	APIServerAdvertiseAddress = "apiserver-advertise-address"
 
-// APIServerBindPort flag sets the port for the API Server to bind to.
-const APIServerBindPort = "apiserver-bind-port"
+	// APIServerBindPort flag sets the port for the API Server to bind to.
+	APIServerBindPort = "apiserver-bind-port"
```

Comment:
let's keep the change of moving all `const` under a single `const (` to another PR.

Reply:
https://github.com/kubernetes/kubeadm/issues/1400

Reply:
thanks